modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsTable/index.tsx

```diff
@@ -114,6 +116,16 @@ const ClaimsTable = () => {
 		pageSize: pageSize.toString(),
 	};
 
+	const claimsChartStatuses = {
+		remaining: 'Remaining',
+		settled: 'Settled',
+	};
+
+	const claimsChartTypes = {
+		settledClaims: 'settledClaims',
+		totalClaims: 'totalClaims',
+	};
+
```

Comment:
I think it will be better to turn these objects into enum.

```suggestion
	enum ClaimsChartStatuses  {
		Remaining: 'Remaining',
		Settled: 'Settled',
	};

	enum ClaimsChartTypes  {
		SettledClaims: 'settledClaims',
		TotalClaims: 'totalClaims',
	};

```

Reply:
Done!